network-api/networkapi/wagtailpages/fields.py

```diff
@@ -49,7 +51,7 @@ class ExtendedYesNoField(models.CharField):
     def __init__(self, *args, **kwargs):
         kwargs["choices"] = self.choice_list
         kwargs["default"] = self.default_choice
-        kwargs["max_length"] = 3
+        kwargs["max_length"] = 4
```

Comment:
Updated so we can use "Good" as an option without abbreviating it. 

I cant quite find why a max length was implemented in the first place 🤔 it does not seem like its being used anywhere, besides making sure that the first item in the `("x", "x")`, is under 3/4 characters.